scripts/vsce-bundled-extension.ts

```diff
@@ -0,0 +1,118 @@
+import {
```

Comment:
This is the script that does the vsce bundling now. Run via npm script. See execution in action on the build-all circleci build

I tried to make it easy to understand both the flow and output.  Let me know if it needs any polish.  See a build [here](https://app.circleci.com/pipelines/github/forcedotcom/salesforcedx-vscode/10631/workflows/8de450a5-d49f-4e63-b725-2092ca893611/jobs/39725): under packaging extensions